flowmachine/flowmachine/features/location/redacted_flows.py

```diff
@@ -5,13 +5,14 @@
 
 from flowmachine.core import Query
 from flowmachine.features import Flows
-from flowmachine.features.location.flows import FlowLike
+from flowmachine.features.location.flows import FlowLike, BaseInOutFlow
 from flowmachine.features.location.redacted_location_metric import (
     RedactedLocationMetric,
 )
 
 
-class RedactedFlows(RedactedLocationMetric, FlowLike, Query):
+# As far as
+class RedactedFlowsLike(RedactedLocationMetric, FlowLike, Query):
```

Comment:
Can't combine these I think because an in/out flow is not flows-like anymore (no from-to), so better two separate classes.

Reply:
Thought I could get away with it as the queries produced are identical - but better to have a RedactedInOutFlows class